### DISCUSSION TITLE
Web Tool failures

### DISCUSSION THREAD

**Discussion by smryania** [NREL/REopt-API-Analysis#151] [1 - General REopt Questions]
The web tool was unable to return utility rates for the site I entered.  Hitting the blue button to solve the instance produced no response.  I tried the sample site with all defaults, which did result in the screen with a blue progress bar, but ended in an error message:
Oops! Please check the errors below.
{"core_urdb.jl_135"=>[" Error occurred HTTP.Exceptions.StatusError(500", " GET", " utility_ratesapi_key2qt5uihpKXdywTj3uMIhBewxY9K4eNjpRje1JUPLversion8formatjsondetailfullgetpage653bf68516b59af52b0b1687", " HTTP.Messages.ResponsennHTTP1.1 500 Internal Server ErrorrnDate Fri", " 19 Apr 2024 153814 GMTrnContentType texthtml charsetUTF8rnContentLength 0rnConnection keepalivernAccessControlAllowOrigin rnAge 0rnExpectCt maxage7776000", " reporturihttpsopenei.orgwikiOpenEIAboutrnStrictTransportSecurity maxage31536000preload", " maxage31536000 preloadrnVia ma4", " i1122w2.compute.internal", " https1.1 apiumbrella (ApacheTrafficServer cMsSf )rnXApiUmbrellaRequestId ceus11kjoqisieqkh940rnXCache MISS from i1122w2.compute.internalrnXContentTypeOptions nosniffrnXRatelimitLimit 1000rnXRatelimitRemaining 958rnXVcapRequestId 42ffbf404a8844c6697c5b1b3b0580fbrnXXssProtection 1 modeblockrnXFrameOptions sameoriginrnrn)"]}

Is it possible to add a web tool status screen?

**Comment by adfarth** [comment]
Hello! The REopt tool utilizes the Utility Rate Database API, which was temporarily down and caused this issue. The issue is now resolved.
